lua/lang/scala.lua

```diff
@@ -0,0 +1,80 @@
+local M = {}
+
+M.config = function()
+  O.lang.scala = {
+    metals = {
+      active = false,
+      server_version = "0.10.5",
+      excluded_packages = {},
+      show_implicit_arguments = false,
+      show_inferred_type = true,
+      status_bar_provider = false,
```

Comment:
I'd probably recommend removing all of these here. Most of them are the defaults like `exluded_packages`,  `show_implicit_argumens`, and `status_bar_provider`. It's a bit odd that the only one you're really setting is `show_inferred_type`. I'd recommend just removing all of these and then letting the user set them.

Reply:
we need to have these somewhere so when the user is trying to change them in their config
they see sth like this and also the default value

![metals](https://user-images.githubusercontent.com/10992695/125996311-51a939e8-a338-4727-ab78-32089e1e34ac.png)

Reply:
@ckipp01 this is a better picture 😅 
<img width="783" alt="Screen Shot 2021-07-16 at 11 33 20 PM" src="https://user-images.githubusercontent.com/10992695/125996633-541dc0e2-3c4d-4c43-aba7-fbde50907fb2.png">